native/linux-clang-cross/build.cake

```diff
@@ -2,31 +2,33 @@ DirectoryPath ROOT_PATH = MakeAbsolute(Directory("../.."));
 
 #load "../../scripts/cake/shared.cake"
 
+string TOOLCHAIN_ARCH = Argument("toolchainArch", EnvironmentVariable("TOOLCHAIN_ARCH"));
+string TOOLCHAIN_ARCH_SHORT = Argument("toolchainArchShort", EnvironmentVariable("TOOLCHAIN_ARCH_SHORT"));
+string TOOLCHAIN_ARCH_TARGET = Argument("toolchainArchTarget", EnvironmentVariable("TOOLCHAIN_ARCH_TARGET"));
```

Comment:
I now define these in the Docker container instead of having it calculated in other places each time we use it.